### PR TITLE
Add gradle artifacts

### PR DESCRIPTION
**Pull request by mezarin**
This PR adds basic Gradle artifacts.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
         <dependency>
             <groupId>org.eclipse.microprofile</groupId>
             <artifactId>microprofile</artifactId>
-            <version>4.0.1</version>
+            <version>4.0</version>
```
Comment (scottkurz) [inline pom.xml]: I think 4.0 has a bug which is why we were using 4.0.1.

File: pom.xml [hunk 2]
```diff
@@ -57,7 +57,7 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
-            <version>5.6.0</version>
+            <version>5.7.2</version>
```
Comment (scottkurz) [inline pom.xml:60]: Is this just the latest version? Or is it supposed to align with a version cached in a stack image?   (If so I wonder how we'd keep the two in sync?)

**Review by scottkurz — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (mezarin) [reply pom.xml]: This change was made to align with the Open Liberty starter application that you can generate. I can change that back to 4.0.1

File: pom.xml (hunk 2 above)
Comment (mezarin) [reply pom.xml:60]: I updated the versions in the stack PR that is currently pending review ... so this aligns with those changes.

**Review by scottkurz — APPROVED** [review]
Thx. LGTM

**Pull request merged**